app/components/AsciidocBlocks/Image.tsx

```diff
@@ -7,20 +7,15 @@
  */
 
 import * as Ariakit from '@ariakit/react'
-import { CaptionedTitle, type AdocTypes } from '@oxide/react-asciidoc'
+import { type Block, type Inline } from '@asciidoctor/core'
+import { Title, useConverterContext, type ImageBlock } from '@oxide/react-asciidoc'
 import { useState } from 'react'
 
-function nodeIsInline(node: AdocTypes.Block | AdocTypes.Inline): node is AdocTypes.Inline {
+function nodeIsInline(node: Block | Inline): node is Inline {
   return node.isInline()
 }
 
-const Image = ({
-  node,
-  hasLightbox = true,
-}: {
-  node: AdocTypes.Block | AdocTypes.Inline
-  hasLightbox?: boolean
-}) => {
+const InlineImage = ({ node }: { node: Block | Inline }) => {
```

Comment:
Inline elements we cannot use the custom renderer unfortunately. So this is formatted like a regular AsciiDoc converter function. Inline images appear in places like table cells.